WordPress/Classes/Utility/BuildInformation/FeatureFlag.swift

```diff
@@ -21,6 +22,8 @@ enum FeatureFlag: Int {
             return true
         case .extractNotifications:
             return BuildConfiguration.current == .localDeveloper
+        case .customDomainForBusinessPlan:
+            return true
```

Comment:
Returning `true` _always_ enabled this feature, thus allowing it to be visible to everyone, including customers. We're not quite ready for that. 😄 

Since this feature is in development, we need to restrict it accordingly. Let's change this to:
`return BuildConfiguration.current == .localDeveloper` 

(`BuildConfiguration.swift` explains the various states a bit.)

Reply:
Done